svghmi/gen_index_xhtml.xslt

```diff
@@ -2750,35 +2756,98 @@
     <xsl:text>BackWidget</xsl:text>
     <xsl:text> extends Widget{
 </xsl:text>
-    <xsl:text>    on_click(evt) {
+    <xsl:text>    frequency = 2;
```

Comment:
Why is frequency needed ?

Reply:
Most of the widgets have it. I used a value from Jump Widget, as Back could be considered as special case of Jump.
I removed it.